apps/google-analytics-4/frontend/src/components/config-screen/NextStepsSection/NextStepsSection.tsx

```diff
@@ -0,0 +1,45 @@
+import HyperLink from 'components/common/HyperLink/HyperLink';
+import { Paragraph, Subheading } from '@contentful/f36-components';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+
+export const NO_CONTENT_TYPES_MSG =
+  'Be sure to save this configuration. Once you have configured a content type, head over to the Content tab, open an entry associated with a configured content type, and view the app in the sidebar.';
+export const CONTENT_TYPES_MSG =
+  'Be sure to save this configuration. Head over to the Content tab, open an entry associated with a configured content type, and view the app in the sidebar.';
+
+interface Props {
+  isContentTypeConfigured?: boolean;
+}
+
+const NextStepsSection = (props: Props) => {
+  const { isContentTypeConfigured } = props;
+  const sdk = useSDK<AppExtensionSDK>();
+
+  const openEntriesList = () => sdk.navigator.openEntriesList();
+
+  return (
+    <>
+      <Subheading marginBottom="spacingXs">View app</Subheading>
```

Comment:
I don't mind "View app" as the section header; maybe another option to consider is "View app on content entry"?

Reply:
oh I like that, will adjust, thank you!